### PR TITLE
Vending Machine Overlay Fix

### PR DESCRIPTION
**Pull request by Geevies**
* Vending machine screens no longer overlay over shutters.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/game/machinery/doors/blast_door.dm [hunk 1]
```diff
@@ -153,6 +153,10 @@
 	if(autoclose)
 		spawn(150)
 			close()
+	var/obj/machinery/vending/V = locate() in get_turf(src)
+	if(V)
+		V.cut_overlays()
+		V.add_screen_overlay()
```
Comment (NonQueueingMatt) [inline code/game/machinery/doors/blast_door.dm:159]: This is very snowflaky. Do what I told you to do on discord instead.

**Comment by NonQueueingMatt** [comment]
i didn't mean to approve this

**Comment by skull132** [comment]
..How are the overlays layered? Isn't this literally just a layering issue?

**Comment by NonQueueingMatt** [comment]
Unfinished. Reopen when ready.

**Pull request closed without merging**